src/Elcodi/Component/Banner/Services/BannerManager.php

```diff
@@ -57,7 +57,7 @@ public function getBannersFromBannerZoneCode($bannerZoneCode, LanguageInterface
     {
         return $this
             ->bannerRepository
-            ->getBannerByZone($bannerZoneCode, $language);
+            ->getBannerByZone((string) $bannerZoneCode, $language);
```

Comment:
Imo where there's an abstraction there should be a "trust level".
If the `bannerRepository` is always trusted, than no problem, everything can be passed to it, but if the `bannerRepository` isn't so "strong" to accept every type and requires a string (based on `BannerManager` docblock) than a string must be passed.

I'm not sure if I explained well what I'm thinking ^^'